packages/entity/src/utils/testing/StubDatabaseAdapter.ts

```diff
@@ -76,13 +76,15 @@ export default class StubDatabaseAdapter<T> extends EntityDatabaseAdapter<T> {
     const bField = objectB[currentOrderBy.columnName];
     switch (currentOrderBy.order) {
       case OrderByOrdering.DESCENDING:
-        return aField > bField
+        // simulate NULLS FIRST for DESC
+        return aField > bField || aField === null
```

Comment:
null is ordered somewhere between -1 and 0 when comparing it against integers. I think it is clearer to do the null checks first (if both are null, return 0, if either one is null, return 1 or -1) and then run the regular comparison checks once we know both values are non-null.

As it stands I think this current implementation breaks down with aField = 1, bField = null.

(worth adding a regression test perhaps)

Reply:
At this point we are not guaranteed `aField !== null`:
```suggestion
        return aField > bField
```